src/quick-start.md

```diff
@@ -58,23 +72,23 @@ spec:
 
 Overview of the attributes of the `PolicyServer` resource:
 
-* `image`: container image name
-* `replicas`: number of desired instances
-* `serviceAccountName` (optional): `policy-server` ServiceAccount. Namespace's `default` ServiceAccount will be used if nothing is provided
-* `env` (optional): `policy-server` environment variables
-* `annotations` (optional): `policy-server` annotations
+| Required | Placeholder         | Description    |
+|:--------:| ------------------- | ----------------------------- |
+| ✅ | `image`  | The name of the container image |
+| ✅ | `replicas`  | The number of desired instances |
+| - | `serviceAccountName` | The name of the `ServiceAccount` to use for the `PolicyServer` deployment. If no value is provided, the `ServiceAccount` from the `default` namespace will be used |
```

Comment:
It is not correct to state that the `ServiceAccount` from the `default` namespace will be used. We are not deploying the controller inside of the `default` namespace.

What happens is, the `default` ServiceAccount of the namespace where the controller is deployed is going to be used.

Reply:
Rephrased as suggested